release_notes/ocp-4-0-release-notes.adoc

```diff
@@ -0,0 +1,778 @@
+[cols="3",options="header"]
+|====
+|Feature |New installer |OperatorHub
+
+|Console and authentication
+|* [x]
+| -
+
+|Monitoring
+|* [x]
+| -
+
+|Over-the-air updates
+|* [x]
+| -
+
+|Machine management
+|* [x]
+| -
+
+|Optional service brokers
+| -
+|* [x]
+
+|Optional {product-title} components
+| -
+|* [x]
+
+|Red Hat product Operators
+| -
+|* [x]
+
+|Red Hat partner Operators
+| -
+|* [x]
+
+|Community Operators
+| -
+|* [x]
+
+|====
+
+[[ocp-4-1-storage]]
+=== Storage
+
+Storage support in {product-title} 4.1 is the same as {product-title} 3.11 with
+the exception of the following available in Technology Preview: EFS (CSI Driver
+handled via Amazon), Manilla provisioner/operator, and Snapshot.
+
+[[ocp-4-1-CSI]]
+==== Container Storage Interface
+
+In {product-title} 4.1, Container Storage Interface (CSI) is now generally
+available.
+
+[[ocp-4-1-scale]]
+=== Scale
+
+[[ocp-4-1-scale-cluster-limits]]
+==== Cluster limits
+
+Updated guidance around
+xref:../scalability_and_performance/planning-your-environment-according-to-object-limits.adoc[Cluster
+Limits] for {product-title} 4.1 is now available. In this release, the pod limit
+on nodes increased from 250 to 500.
+
+Use the link:https://access.redhat.com/labs/ocplimitscalculator/[{product-title}
+Limit Calculator] to estimate cluster limits for your environment.
+
+[[ocp-4-1-node-tuning-operator]]
+==== Node Tuning Operator
+
+The
+xref:../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[Node
+Tuning Operator] is now part of a standard {product-title} installation in
+version 4.1 and later.
+
+The Node Tuning Operator helps you manage node-level tuning by orchestrating the
+tuned daemon. The majority of high-performance applications require some level
+of kernel tuning. The Node Tuning Operator provides a unified management
+interface to users of node-level sysctls and more flexibility to add custom
+tuning, which is currently a Technology Preview feature, specified by user
+needs. The Operator manages the containerized tuned daemon for OpenShift
+Container Platform as a Kubernetes DaemonSet. It ensures the custom tuning
+specification is passed to all containerized tuned daemons running in the
+cluster in the format that the daemons understand. The daemons run on all nodes
+in the cluster, one per node.
+
+[[ocp-4-1-metrics-and-logging]]
+=== Metrics and logging
+
+[[ocp-4-1-metering]]
+=== Metering
+
+[[ocp-4-1-metering-and-chargeback]]
+==== Metering and chargeback (Technology Preview)
+
+Metering and chargeback is a Technology Preview feature in {product-title} 4.1.
+
+The Operator Metering portion of the Operator framework gives administrators
+insight into usage for specific Operators. Usage metrics are unique to each
+Operator; metering watches custom resources and collects data, and performs
+calculations and reports on that data back to the administrator.
+
+[[ocp-4-1-cluster-monitoring]]
+==== Cluster monitoring
```

Comment:
cluster monitoring should probably not be under "metering" instead should be it's own `===` section.